.github/workflows/review-round-dispatch.yml

```
@@ -0,0 +1,68 @@
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review, labeled, unlabeled]
+  pull_request_review:
+    types: [submitted, edited, dismissed]
+  pull_request_review_comment:
+    types: [created, edited, deleted]
+  # Resolving the last round thread in the UI, or a bot completing only a check
+  # run, must also wake the gate — otherwise the ledger will not advance until an
+  # unrelated push/comment/label fires (issue #1992 P2).
+  pull_request_review_thread:
```

actionlint (1.7.12)

[error] unknown Webhook event "pull_request_review_thread". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)